src/components/SourceTabs.js

```diff
@@ -287,12 +282,27 @@ const SourceTabs = React.createClass({
   },
 
   render() {
+    let endPaneCollapseButton;
+
+    if (this.props.horizontal) {
+      endPaneCollapseButton = PaneCollapseButton({
+        position: "end",
+        collapsed: !this.props.endPanelCollapsed,
+        handleClick: this.props.togglePane,
+        horizontal: this.props.horizontal
+      });
+    }
+
     return dom.div({ className: "source-header" },
-      this.renderToggleButton("start", !this.props.startPanelCollapsed),
+      PaneCollapseButton({
```

Comment:
@jasonLaster Yeah, the button was refactored for that and would go in the `CommandBar` component for the bottom pane.